programs/sll_delete_middle_node.py

```diff
@@ -0,0 +1,37 @@
+# in a single linked list delete a node in de middle, not the first, not de last, not necesary the exact middle
+# You only receive the Node to delete, not de sll
+
+
+from single_linked_lists import Node
+
+
+# I have not element before I want to delete, I will replace values with next values  and delete next
+def delete_element(element: Node):
+    print("Hola")
+    element.data = element.next.data
+    element.next = element.next.next
+
+
+h = Node.list_builder()
+print(h)
+
+k = int(input("Which element in the middle do You want to delete?: "))
+
+found = 0
+n = h
+# Search element to delete
+while n and n.next:
```

Comment:
Your solution works for first node too.
```suggestion
while n:
```